homeworks/nikolay.krasniuk_Kingstar22/5-rxjs/style.css

```diff
@@ -0,0 +1,211 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  background: #C4D9FF;
+}
+
+.main {
+  max-width: 50%;
+  margin: 0 auto;
+  box-shadow: 0 5px 10px rgba(35, 36, 40, .5);
+  background: #F3F7FE;
+}
+
+.main__header {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  position: sticky;
+  top: -100px;
+  height: 60px;
+  width: 100%;
+  background: #44536B;
+  padding: 15px 15px 15px 0;
+  transition: top .5s;
+}
+
+.main__wrapper {
+  display: flex;
+  justify-content: space-between;
+}
+
+.main__header-logo {
+  margin: 0;
+  font-size: 30px;
+  color: #F3F7FE;
+  cursor: pointer;
+  padding-left: 15px;
+}
+
+.main__header-burger {
+  display: none;
+}
+
+.main__nav-menu {
+  display: flex;
+  flex-grow: 1;
+  align-self: self-end;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  margin-bottom: 3px;
+}
+
+.main__nav-menu-link {
+  font-size: 16px;
+  padding-left: 15px;
+  color: #F3F7FE;
+  text-decoration: none;
+}
+
+.main__header-btn, .main__content-btn {
+  border-radius: 5px;
+  border: 1px solid #FFD8CC;
+  font-size: 18px;
+  font-weight: 500;
+  background-color: #FFD8CC;
+  color: #5E4040;
+  cursor: pointer;
+}
+
+.main__content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 15px;
+  line-height: 25px;
+}
+
+.main__content-text {
+  margin: 0;
+  text-align: justify;
+}
+
+
+.main__content-btn {
+  margin: 0 auto;
+}
+
+.active {
+  top: 0;
+}
+
+@media (max-width: 890px) {
```

Comment:
It's better to use existing sizes for media queries. I consider using 768px here.

Reply:
done